gsy_framework/community_datasheet/row_converters.py

```diff
@@ -23,10 +23,17 @@
     CommunityMembersSheetHeaderOptional,
 )
 from gsy_framework.constants_limits import ConstSettings
-from gsy_framework.utils import use_default_if_null
+from gsy_framework.utils import use_default_if_null, convert_string_to_boolean
+from gsy_framework.enums import CoefficientAlgorithm
 
 logger = logging.getLogger(__name__)
 
+# this is in order to expose better naming to the user
+ADVANCED_SETTINGS_FIELD_MAPPING = {
+    "intracommunity_rate": "intracommunity_rate_base_eur",
+    "operational_hours_of_delay": "scm_cn_hours_of_delay",
```

Comment:
Operational hours of delay is a so much self-evident variable than scm_cn_hours_of_delay 😅. The intracommunity_rate is only valid for communities with fixed price right? and just need to calculate the energy bill from that. It should be optional since that is not the case for the communities we have now.

Reply:
> Operational hours of delay is a so much self-evident variable than scm_cn_hours_of_delay 😅.

yeah. this is why this mapping exists. How about adding a ticket for renaming `scm_cn_hours_of_delay` to `operational_hours_of_delay` ?

> The intracommunity_rate is only valid for communities with fixed price right? and just need to calculate the energy bill from that. 

Not sure what you mean. The `intracommunity_rate_base_eur` is already used on the gsy-e side. So we need to somehow pass it there. I would not distinguish between optional and required parameters here, as they are all in a separate Excel Tab which we could communicate to the user to only touch with our supervision. What do you think?

Reply:
Telling javier and jose that they can only edit the Advanced Tab with our supervision is IMO a big turn off. If the intracommunity_rate_base_eur is a property that should be set if the community managers want a fixed price for the community then it should be optional, since that may not be the case.

Reply:
changing scm_cn_hours_of_delay  to operational_hours_of_delay is a good move, and we can create a ticket for that and next time we change something in that table we pitch in that ticket too.

Reply:
> Telling javier and jose that they can only edit the Advanced Tab with our supervision is IMO a big turn off

I am fine with showing all the options to him and pre-populate them already in the default CDS.
If you think that this is too overwhelming, then we need to review the list of advanced settings to be exposed by default here.

Let me show how the Advanced Settings tab is currently looking:
<img width="353" alt="image" src="https://github.com/user-attachments/assets/bd4df305-48b8-44f7-bd1a-1cecafd6d32d">

Reply:
For now we decided to keep these settings in the CDS always visible to the user.